Source/CarthageKit/Git.swift

```diff
@@ -57,6 +57,30 @@ public struct GitURL: Equatable {
 		}
 	}
 
+	/// Amount of time before a git repository is fetched again. Defaults to 1 minute
+	public static var fetchCacheInterval: NSTimeInterval = 60.0
```

Comment:
I put it here because it is the internal implementation detail of the non static 'needsFetch' and 'updateFetchTime', but I can move it outside of the struct